IDE/MonoDevelop/MonoDevelop.MonoGame.iOS/templates/Shared/Activity1.cs

```diff
@@ -9,8 +9,9 @@
 using Android.OS;
 
 using Microsoft.Xna.Framework;
+using ${ProjectName};
 
-namespace ${Namespace}
+namespace ${ProjectName}.Droid
```

Comment:
Why the `.Droid` addition to the namespace?  It just necessitates the additional `using` statement above it, and "Droid" is not an abbreviation that Google approve of for "Android".

Also, a project name might not be a valid namespace if I remember correctly.